### PR TITLE
The Witness: Fix newlines in Witness option tooltips

### PR DESCRIPTION
**Pull request by NewSoupVi**
I was made aware that newlines in option tooltips are actually applied on the Webhost.
Some of the Witness option tooltips look strange as a result.

I am very, very upset that I have to break Pep8 guidelines (line length) for my option tooltips to look good.

Also, changed some outdated hints.

### CODE REVIEW

**Review by ScipioWright — APPROVED** [review]
proper formatting on docstrings
didn't forget some
bruh

File: worlds/witness/options.py [hunk 1]
```diff
@@ -8,19 +8,23 @@
...
 
 class EarlyCaves(Choice):
-    """Adds an item that opens the Caves Shortcuts to Swamp and Mountain,
-    allowing early access to the Caves even if you are not playing a remote Door Shuffle mode.
-    You can either add this item to the pool to be found on one of your randomized checks,
-    or you can outright start with it and have immediate access to the Caves.
-    If you choose "add_to_pool" and you are already playing a remote Door Shuffle mode, this setting will do nothing."""
+    """
+    Adds an item that opens the Caves Shortcuts to Swamp and Mountain, allowing early access to the Caves even if you are not playing a remote Door Shuffle mode.
+    You can either add this item to the pool to be found on one of your randomized checks, or you can outright start with it and have immediate access to the Caves.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:24]: This implied that it was forced local, this might be a better wording:
```suggestion
    You can either add this item to the pool to be found in the multiworld, or you can outright start with it and have immediate access to the Caves.
```

File: worlds/witness/options.py [hunk 2]
```diff
@@ -50,10 +58,12 @@ class ShuffleLasers(Choice):
 
 
 class ShuffleDoors(Choice):
-    """If on, opening doors, moving bridges etc. will require a "key".
+    """
+    If on, opening doors, moving bridges etc. will require a "key".
     If set to "panels", the panel on the door will be locked until receiving its corresponding key.
     If set to "doors", the door will open immediately upon receiving its key. Door panels are added as location checks.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:64]: I wonder if this line should be changed a bit, while it's still true that "door panels are added as location checks", it means a lot less now

File: worlds/witness/options.py [hunk 3]
```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
 
 
 class DoorGroupings(Choice):
-    """If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
-    If set to "regional", all doors in the same general region will open at once with a single key,
-    reducing the amount of door items and complexity."""
+    """
+    If set to "none", there will be one key for every door, resulting in up to 120 keys being added to the item pool.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:76]: I think this number is wrong now, will test later
```suggestion
    If set to "none", there will be one key for each door, resulting in up to 120 keys being added to the item pool.
```

File: worlds/witness/options.py [hunk 4]
```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
...
+    """
     display_name = "Door Groupings"
     option_off = 0
     option_regional = 1
 
 
 class ShuffleBoat(DefaultOnToggle):
-    """If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat."""
+    """
+    If set, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:86]: You used "on" elsewhere
```suggestion
    If on, adds a "Boat" item to the item pool. Before receiving this item, you will not be able to use the boat.
```

File: worlds/witness/options.py [hunk 5]
```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
...
+    """
     display_name = "Shuffle Boat"
 
 
 class ShuffleDiscardedPanels(Toggle):
-    """Add Discarded Panels into the location pool.
-    Solving certain Discarded Panels may still be necessary to beat the game, even if this is off - The main example
-    of this being the alternate activation triggers in disable_non_randomized."""
+    """
+    Add Discarded Panels into the location pool.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:93]:
```suggestion
    Adds Discarded Panels into the location pool.
```

File: worlds/witness/options.py [hunk 6]
```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
...
+
+    Solving certain Discarded Panels may still be necessary to beat the game, even if this is off - The main example of this being the alternate activation triggers in disable_non_randomized.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:95]: This sentence makes more sense flipped around, changes away from snake_case option name because you don't usually use that
```suggestion
    Even if this is off, solving certain Discarded Panels may still be necessary to beat the game - The main example of this being the alternate activation triggers in "Disable non randomized puzzles".
```

File: worlds/witness/options.py [hunk 7]
```diff
@@ -62,29 +72,36 @@ class ShuffleDoors(Choice):
...
+    """
 
     display_name = "Shuffle Discarded Panels"
 
 
 class ShuffleVaultBoxes(Toggle):
-    """Add Vault Boxes to the location pool."""
+    """
+    Add Vault Boxes to the location pool.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:103]:
```suggestion
    Adds Vault Boxes to the location pool.
```

File: worlds/witness/options.py [hunk 8]
```diff
@@ -93,6 +110,7 @@ class ShuffleEnvironmentalPuzzles(Choice):
     Add Environmental/Obelisk Puzzles into the location pool.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:110]:
```suggestion
    Adds Environmental/Obelisk Puzzles into the location pool.
```

File: worlds/witness/options.py [hunk 9]
```diff
@@ -93,6 +110,7 @@ class ShuffleEnvironmentalPuzzles(Choice):
     Add Environmental/Obelisk Puzzles into the location pool.
     In "individual", every Environmental Puzzle sends an item.
     In "obelisk_sides", completing every puzzle on one side of an Obelisk sends an item.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:112]:
```suggestion
    If set to "Obelisk Sides", completing every puzzle on one side of an Obelisk sends an item.
```

File: worlds/witness/options.py [hunk 10]
```diff
@@ -93,6 +110,7 @@ class ShuffleEnvironmentalPuzzles(Choice):
     Add Environmental/Obelisk Puzzles into the location pool.
     In "individual", every Environmental Puzzle sends an item.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:111]:
```suggestion
    If set to "individual", every Environmental Puzzle sends an item.
```

File: worlds/witness/options.py [hunk 11]
```diff
@@ -102,7 +120,9 @@ class ShuffleEnvironmentalPuzzles(Choice):
 
 
 class ShuffleDog(Toggle):
-    """Add petting the Town dog into the location pool."""
+    """
+    Add petting the Town dog into the location pool.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:124]:
```suggestion
    Adds petting the Town dog into the location pool.
```

File: worlds/witness/options.py [hunk 12]
```diff
@@ -123,26 +143,33 @@ class EnvironmentalPuzzlesDifficulty(Choice):
 class ObeliskKeys(DefaultOnToggle):
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:143]: I can't actually comment on the EP Difficulty section but you could change `On "X"` to `If set to "X"`

File: worlds/witness/options.py [hunk 13]
```diff
@@ -123,26 +143,33 @@ class EnvironmentalPuzzlesDifficulty(Choice):
...
 
 class ShufflePostgame(Toggle):
-    """Adds locations into the pool that are guaranteed to become accessible after or at the same time as your goal.
-    Use this if you don't play with release on victory. IMPORTANT NOTE: The possibility of your second
-    "Progressive Dots" showing up in the Caves is ignored, they will still be considered "postgame" in base settings."""
+    """
+    Adds locations into the pool that are guaranteed to become accessible after or at the same time as your goal.
+    Use this if you don't play with release on victory.
+
+    IMPORTANT NOTE: The possibility of your second "Progressive Dots" showing up in the Caves is ignored, they will still be considered "postgame" in base settings.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:157]: I wonder if this note can be removed, since there's honestly a few other things being accounted for. Did people ask about this before it was added to the tooltip?

File: worlds/witness/options.py [hunk 14]
```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
...
     option_none = 2
 
 
 class MountainLasers(Range):
-    """Sets the amount of lasers required to enter the Mountain.
-    If set to a higher amount than 7, the mountaintop box will be slightly rotated to make it possible to solve without
-    the hatch being opened.
-    This change will also be applied logically to the long solution ("Challenge Lasers" setting)."""
+    """
+    Sets the amount of lasers required to enter the Mountain.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:192]: Number should be used instead of amount here
```suggestion
    Sets the number of lasers required to enter the Mountain.
```

File: worlds/witness/options.py [hunk 15]
```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
...
+    If set to a higher amount than 7, the mountaintop box will be slightly rotated to make it possible to solve without the hatch being opened.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:193]:
```suggestion
    If set to a higher number than 7, the mountaintop box will be slightly rotated to make it possible to solve without the hatch being opened.
```

File: worlds/witness/options.py [hunk 16]
```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
...
     display_name = "Required Lasers for Mountain Entry"
     range_start = 1
     range_end = 11
     default = 7
 
 
 class ChallengeLasers(Range):
-    """Sets the amount of beams required to enter the Caves through the Mountain Bottom Floor Discard."""
+    """
+    Sets the amount of beams required to enter the Caves through the Mountain Bottom Floor Discard.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:204]: It may be good to explicitly state that this also impacts the Challenge Timer Panel
```suggestion
    Sets the number of lasers required to enter the Caves through the Mountain Bottom Floor Discard.
```

File: worlds/witness/options.py [hunk 17]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
...
+    Only applies if laser shuffle is enabled.
+    """
     display_name = "Laser Hints"
 
 
 class DeathLink(Toggle):
-    """If on: Whenever you fail a puzzle (with some exceptions), everyone who is also on Death Link dies.
-    The effect of a "death" in The Witness is a Bonk Trap."""
+    """
+    If on: Whenever you fail a puzzle (with some exceptions), everyone who is also on Death Link dies.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:294]: Consistency
```suggestion
    If on, Whenever you fail a puzzle (with some exceptions), everyone who is also on Death Link dies.
```

File: worlds/witness/options.py [hunk 18]
```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
...
     range_start = 1
     range_end = 11
     default = 11
 
 
 class ElevatorsComeToYou(Toggle):
-    """If true, the Quarry Elevator, Bunker Elevator and Swamp Long Bridge will "come to you" if you approach them.
-    This does actually affect logic as it allows unintended backwards / early access into these areas."""
+    """
+    If true, the Quarry Elevator, Bunker Elevator and Swamp Long Bridge will "come to you" if you approach them.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:214]:
```suggestion
    If on, the Quarry Elevator, Bunker Elevator and Swamp Long Bridge will "come to you" if you approach them.
```

File: worlds/witness/options.py [hunk 19]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
...
     range_start = 0
     range_end = 49
     default = 12
 
 
 class AreaHintPercentage(Range):
-    """There are two types of hints for The Witness.
+    """
+    There are two types of hints for The Witness.
     "Location hints" hint one location in your world / containing an item for your world.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:274]: The `/` is pretty hard to parse here, I think spelling it out explicitly makes more sense
```suggestion
    "Location hints" hint one location in your world or one location containing an item for your world.
```

File: worlds/witness/options.py [hunk 20]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
...
-    "Area hints" will tell you some general info about the items you can find in one of the
-    main geographic areas on the island.
-    Use this option to specify how many of your hints you want to be area hints. The rest will be location hints."""
+    "Area hints" will tell you some general info about the items you can find in one of the main geographic areas on the island.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:275]: Tense consistency
```suggestion
    "Area hints" tell you some general info about the items you can find in one of the main geographic areas on the island.
```

File: worlds/witness/options.py [hunk 21]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
...
     range_end = 100
     default = 33
 
 
 class LaserHints(Toggle):
-    """If on, lasers will tell you where their items are if you walk close to them in-game.
-    Only applies if laser shuffle is enabled."""
+    """
+    If on, lasers will tell you where their items are if you walk close to them in-game.
+    Only applies if laser shuffle is enabled.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:287]: How you format these is somewhat inconsistent, but they are always capitalized
```suggestion
    Only applies if Laser Shuffle is enabled.
```

File: worlds/witness/options.py (hunk 17 above)
Comment (Exempt-Medic) [inline worlds/witness/options.py:294]: Explicitly mentioning the self-death might be good:
```suggestion
    If on: Whenever you fail a puzzle (with some exceptions), you and everyone who is also on Death Link dies.
```

File: worlds/witness/options.py [hunk 22]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
...
+    The effect of a "death" in The Witness is a Bonk Trap.
+    """
     display_name = "Death Link"
 
 
 class DeathLinkAmnesty(Range):
-    """Number of panel fails to allow before sending a death through Death Link.
-    0 means every panel fail will send a death, 1 means every other panel fail will send a death, etc."""
+    """
+    Number of panel fails to allow before sending a death through Death Link.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:302]: Could also be "How many panel fails"
```suggestion
    The number of panel fails to allow before sending a death through Death Link.
```

File: worlds/witness/options.py [hunk 23]
```diff
@@ -210,50 +248,60 @@ class TrapWeights(OptionDict):
 
 
 class PuzzleSkipAmount(Range):
-    """Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
-    Works on most panels in the game - The only big exception is The Challenge."""
+    """
+    Adds this number of Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
```
Comment (Exempt-Medic) [inline worlds/witness/options.py:252]: idk, could be "many", maybe not. There's a lot of ways to word this, how it is now is fine though
```suggestion
    Adds this many Puzzle Skips into the pool, if there is room. Puzzle Skips let you skip one panel.
```

**Review by Exempt-Medic — CHANGES_REQUESTED** [review]

File: worlds/witness/options.py (hunk 2 above)
Comment (NewSoupVi) [reply worlds/witness/options.py:64]: Could just go with "All"? And then maybe some sort of... "including ones that are usually not location checks?" Idk

File: worlds/witness/options.py (hunk 13 above)
Comment (NewSoupVi) [reply worlds/witness/options.py]: That's probably true, it's one of those things you probably wouldn't have ever considered if the option tooltip didn't point it out to you, I can probably just remove that

File: worlds/witness/options.py (hunk 16 above)
Comment (NewSoupVi) [reply worlds/witness/options.py:202]: How would you word that?

File: worlds/witness/options.py (hunk 2 above)
Comment (Exempt-Medic) [reply worlds/witness/options.py:64]: Maybe just "Some additional door panels" ?

File: worlds/witness/options.py (hunk 16 above)
Comment (Exempt-Medic) [reply worlds/witness/options.py:202]: Probably just add "and to start The Challenge" at the end?

**Review by Exempt-Medic — APPROVED** [review]
Comments were addressed. LGTM

**Review by Berserker66 — APPROVED** [review]
